polaris-core/src/main/java/org/apache/polaris/core/storage/StorageLocation.java

```diff
@@ -93,6 +93,9 @@ public boolean isChildOf(StorageLocation potentialParent) {
     } else {
       String slashTerminatedLocation = ensureTrailingSlash(this.location);
       String slashTerminatedParentLocation = ensureTrailingSlash(potentialParent.location);
+      if (slashTerminatedLocation.startsWith("/") && slashTerminatedParentLocation.startsWith("file:///")) {
+        slashTerminatedLocation = "file://" + slashTerminatedLocation;
```

Comment:
I recommend standardizing the URI schema to `file:///` instead of `file://`. The `file:///` format is the correct URI schema, whereas `file://path` is not. To make this consistent and maintainable, we could define a constant like this:
```
public static final String FILE_SCHEME = "file:///";
```
Additionally, could we add a test to ensure this change is verified and behaves as expected?

Reply:
+1, it should definitely be `file:///` not `file://`.

However the  `:///` is [not part of the scheme](https://datatracker.ietf.org/doc/rfc8089/) so that specific constant might be confusing.

Reply:
Make sense. We could give a different constant name, like `FILE_SCHEMA_PATH_PRFIX`, or `LOCAL_PATH_PREFIX`.

Reply:
Thanks for the comments updated for prefix file:/// and testing